core/src/broadcast_stage/broadcast_duplicates_run.rs

```diff
@@ -198,19 +204,22 @@ impl BroadcastRun for BroadcastDuplicatesRun {
 
         // Special handling of last shred to cause partition
         if let Some((original_last_data_shred, partition_last_data_shred)) = last_shreds {
+            self.original_last_data_shreds.extend(
+                original_last_data_shred
+                    .iter()
+                    .map(|shred| shred.payload.clone()),
```

Comment:
Can you use the signature here instead of the payload?

Reply:
done